src/main/java/org/traccar/protocol/TotemProtocolEncoder.java

```diff
@@ -19,37 +19,49 @@
 import org.traccar.StringProtocolEncoder;
 import org.traccar.model.Command;
 import org.traccar.Protocol;
+import org.traccar.helper.Checksum;
 
 public class TotemProtocolEncoder extends StringProtocolEncoder {
 
     public TotemProtocolEncoder(Protocol protocol) {
         super(protocol);
     }
 
-    @Override
-    protected Object encodeCommand(Command command) {
-
-        initDevicePassword(command, "000000");
-
+    public static String formatContent(Command command) {
         switch (command.getType()) {
             case Command.TYPE_CUSTOM:
-                return formatCommand(command, "*%s,%s#", Command.KEY_DEVICE_PASSWORD, Command.KEY_DATA);
+                return String.format("%s,%s",
+                                     command.getAttributes().get(Command.KEY_DEVICE_PASSWORD),
+                                     command.getAttributes().get(Command.KEY_DATA)
+                                    );
             case Command.TYPE_REBOOT_DEVICE:
-                return formatCommand(command, "*%s,006#", Command.KEY_DEVICE_PASSWORD);
+                return String.format("%s,006", command.getAttributes().get(Command.KEY_DEVICE_PASSWORD));
```

Comment:
Why do we need to change these things?

Reply:
Because if we make formatContent function static, then we cannot use formatCommand unfortunately.
That's the issue discussed here: https://github.com/traccar/traccar/pull/4790#discussion_r778360037
So this was the only way I knew how to implement the proposed change/solution.

Reply:
Ah I remember now. It's been a while.

Reply:
Yeah, life moves fast and I did not have time to setup a test instance to test this until today.